common/lib/util/utils.js

```diff
@@ -129,10 +129,16 @@ var Utils = (function() {
 	 * of another constructor
 	 * See node.js util.inherits
 	 */
-	Utils.inherits = (typeof(require) === 'function' && require('util') && require('util').inherits) || function(ctor, superCtor) {
+
+	Utils.inherits = function(ctor, superCtor) {
 		ctor.super_ = superCtor;
 		ctor.prototype = Utils.prototypicalClone(superCtor.prototype, { constructor: ctor });
 	};
+	try {
+        if (typeof(require) === 'function' && require('util') && require('util').inherits) {
+        	Utils.inherits = require('util').inherits;
+		}
+    } catch (err) {  /* Do Nothing. */ }
```

Comment:
The issue is that both `NativeScript` and `Node` both will throw an exception if a module does not exist, when you try to require it. 

 In this case this code assumes that if `require` exists then the module "util" exists.   It doesn't on all platforms.   (I'm not sure what react-native does here, or maybe a util module exists in RN?)    So since I have no idea how RN works, the simplest solution to this "issue" is to catch an expected error on platforms that might not have a `util` module, and do nothing, as `inherits` by default gets set first, then overrode by the `util` if it exists.